LexisNexis.js

```diff
@@ -3,13 +3,14 @@
 	"label": "LexisNexis",
 	"creator": "Philipp Zumstein",
 	"target": "^https?://[^/]*lexis-?nexis\\.com",
+	"targetAll": "^https?://[^/]*lexis-?nexis\\.com",
```

Comment:
@dstillman Is this the correct use of this new parameter?

Reply:
A couple things:

1) It's possible that they've changed their site since https://github.com/zotero/zotero/issues/1021, or we may have just been mistaken about Lexis requiring it. What I'm seeing now is a search-results iframe on the same domain, in which case this parameter isn't actually required. The translator could just access the iframe from the root document.

2) We could still use targetAll — which would require fewer changes to the translator, if it's already written to match the iframe doc — but then it should theoretically use a more restrictive targetAll to avoid extra detectWeb() calls. (That was [the reason](https://github.com/zotero/zotero/issues/1021#issuecomment-243371694) targetAll wasn't just a boolean.)

If we don't want to bother rewriting it to access the iframe from the root doc and there's any doubt about URL patterns used for search results, this is fine — it just shouldn't be considered a model use of the parameter.

As a side note, it seems like LexisNexis Academic is being replaced by Nexis Uni (appearing on advance.lexis.com), which doesn't use iframes (and which we don't currently support).

Reply:
To 1: But I see frames inside the iframe:

![frames-inside-iframe](https://user-images.githubusercontent.com/5199995/39418613-a9a2548c-4c5b-11e8-8a5c-fdf40b7c6a6f.jpg)

Does the `targetAll` parameter just needs to match the addresses of the frames and not the iframe? That would be easy to add some more of the urls here to make it more specific.

> As a side note, it seems like LexisNexis Academic is being replaced by Nexis Uni (appearing on advance.lexis.com), which doesn't use iframes (and which we don't currently support).

AFAIK it is complicated and also the older looking LexisNexis Academics has some advantages, cf. the urls in http://dbis.uni-regensburg.de//detail.php?bib_id=ubman&colors=&ocolors=&lett=fs&tid=0&titel_id=1670 (in German).

Reply:
> But I see frames inside the iframe

Wow. Yeah, I have no idea whether the root doc can access a frame inside an iframe…

> Does the targetAll parameter just needs to match the addresses of the frames and not the iframe?

I can't say that we tested frames inside iframes, but yes, in theory, it just needs to match the URL of the (i)frame where detectWeb needs to run.

For GMU, I don't think you can even get to Academic anymore from the library site — all references now send you to Nexis Uni (and if you load Academic manually, they tell you your institution has upgraded and suggest switching over).